.github/workflows/cd.yml

```diff
@@ -26,140 +26,93 @@ jobs:
-              TOOLCHAIN: stable,
-              TARGET: i686-unknown-linux-musl,
-              NPM_PUBLISH: false,
-              PYPI_PUBLISH: true,
-            }
-          - {
-              NAME: linux-arm64-glibc,
-              OS: ubuntu-22.04,
-              TOOLCHAIN: stable,
-              TARGET: aarch64-unknown-linux-gnu,
-              NPM_PUBLISH: true,
-              PYPI_PUBLISH: true,
-            }
-          - {
-              NAME: linux-arm64-musl,
-              OS: ubuntu-22.04,
-              TOOLCHAIN: stable,
-              TARGET: aarch64-unknown-linux-musl,
-              NPM_PUBLISH: false,
-              PYPI_PUBLISH: true,
-            }
-          - {
-              NAME: win32-x64-mingw,
-              OS: windows-2022,
-              TOOLCHAIN: stable,
-              TARGET: x86_64-pc-windows-gnu,
-              NPM_PUBLISH: false,
-              PYPI_PUBLISH: false,
-            }
-          - {
-              NAME: win32-x64-msvc,
-              OS: windows-2022,
-              TOOLCHAIN: stable,
-              TARGET: x86_64-pc-windows-msvc,
-              NPM_PUBLISH: true,
-              PYPI_PUBLISH: true,
-            }
-          - {
-              NAME: win32-x86-msvc,
-              OS: windows-2022,
-              TOOLCHAIN: stable,
-              TARGET: i686-pc-windows-msvc,
-              NPM_PUBLISH: false,
-              PYPI_PUBLISH: true,
-            }
-          - {
-              NAME: win32-arm64-msvc,
-              OS: windows-2022,
-              TOOLCHAIN: stable,
-              TARGET: aarch64-pc-windows-msvc,
-              NPM_PUBLISH: true,
-              PYPI_PUBLISH: false,
-            }
-          - {
-              NAME: darwin-x64,
-              OS: macos-14,
-              TOOLCHAIN: stable,
-              TARGET: x86_64-apple-darwin,
-              NPM_PUBLISH: true,
-              PYPI_PUBLISH: true,
-            }
-          - {
-              NAME: darwin-arm64,
-              OS: macos-14,
-              TOOLCHAIN: stable,
-              TARGET: aarch64-apple-darwin,
-              NPM_PUBLISH: true,
-              PYPI_PUBLISH: true,
-            }
+        include:
+          - name: linux-x64-glibc
+            os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+            publish_npm: true
+            publish_pypi: true
+          - name: linux-x64-musl
+            os: ubuntu-latest
+            target: x86_64-unknown-linux-musl
+            publish_npm: false
+            publish_pypi: true
+            install_musl: true
+          - name: linux-x86-glibc
+            os: ubuntu-latest
+            target: i686-unknown-linux-gnu
+            publish_npm: false
+            publish_pypi: false
+          - name: linux-x86-musl
+            os: ubuntu-latest
+            target: i686-unknown-linux-musl
+            publish_npm: false
+            publish_pypi: true
+            install_musl: true
+          - name: linux-arm64-glibc
```

Comment:
See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#standard-github-hosted-runners-for-public-repositories for available runners